configuration/parse.go

```diff
@@ -91,6 +109,32 @@ Options:
 	return config, nil
 }
 
+func loadConfigFromFile(configLocation string, config *Configuration) error {
```

Comment:
These two functions are essentially the same. One way to avoid this is to change the signature to allow to pass in a `config *interface{}` which would allow me to do what I am doing with one method, but it becomes INFINITELY reusable at that point. That should be fine since the function is private, but we could also just simplify config a lot to accomplish one function, by combining the configuration structs, and working from there.